benchmarks/triton_kernels_benchmark/flex_attention_benchmark_custom_masks.py

```diff
@@ -74,8 +74,8 @@ def alibi_functional(score, _, h, q_idx, kv_idx):
            for mask in ['NATTEN', 'Alibi']
            for mode in [os.getenv('FA_KERNEL_MODE', 'fwd')]],
         line_arg='provider',
-        line_vals=['triton', 'onednn'],
-        line_names=['Triton', 'OneDNN'],
+        line_vals=['triton'] + ['onednn'] if '580' not in torch.xpu.get_device_name() else [],
+        line_names=['Triton'] + ['OneDNN'] if '580' not in torch.xpu.get_device_name() else [],
```

Comment:
If `580` in `torch.xpu.get_device_name()` then `line_vals` and `line_names` are empty lists. To fix:
```suggestion
        line_vals=['triton'] + (['onednn'] if '580' not in torch.xpu.get_device_name() else []),
        line_names=['Triton'] + (['OneDNN'] if '580' not in torch.xpu.get_device_name() else []),
```